cmd/gossamer/config_test.go

```diff
@@ -1302,3 +1307,74 @@ func Test_setLogConfig(t *testing.T) {
 		})
 	}
 }
+
+// newTestGenesisRawFile returns a test genesis file using "gssmr" raw data
+func newTestGenesisRawFile(t *testing.T, cfg *dot.Config) (filename string) {
+	filename = filepath.Join(t.TempDir(), "genesis.json")
+
+	fp := utils.GetGssmrGenesisRawPath()
+
+	gssmrGen, err := genesis.NewGenesisFromJSONRaw(fp)
+	require.Nil(t, err)
+
+	gen := &genesis.Genesis{
+		Name:       cfg.Global.Name,
+		ID:         cfg.Global.ID,
+		Bootnodes:  cfg.Network.Bootnodes,
+		ProtocolID: cfg.Network.ProtocolID,
+		Genesis:    gssmrGen.GenesisFields(),
+	}
+
+	b, err := json.Marshal(gen)
+	require.Nil(t, err)
+
+	err = os.WriteFile(filename, b, os.ModePerm)
+	require.NoError(t, err)
+
+	return filename
+}
+
+// newTestGenesisAndRuntime create a new test runtime and a new test genesis
+// file with the test runtime stored in raw data and returns the genesis file
+func newTestGenesisAndRuntime(t *testing.T) (filename string) {
+	_ = wasmer.NewTestInstance(t, runtime.NODE_RUNTIME)
```

Comment:
I'm not exactly sure, I copied this from dot.NewTestGenesisAndRuntime so that I could make the function in dot not exported.  I think this builds genesis files that include runtime code for tests that need that.

Reply:
Added a suggestion for it, resolving this in favor of the suggestion.